### PR TITLE
Fixes lerna publish

### PR DESCRIPTION
**Pull request by jcchavezs**
This PR includes a step in the publish process that cleans up all old builds and avoids old code to show up in the `lib` folder of the node modules. This not only reduces the amount of files in the `lib` but also avoids confusions when debugging. Notice that we already used `lerna clean` but that only cleans the `node-modules` folder.

Important thing happens here: https://github.com/openzipkin/zipkin-js/pull/436/files#diff-cce1d632ab9e7d2789dc6ade2f34650cR8

Thanks a lot to @ca057

Ping @adriancole

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,15 +33,11 @@
     "@babel/preset-typescript": "^7.3.3",
     "@babel/register": "^7.3.3",
     "@types/chai": "^4.1.7",
-    "@types/mocha": "^5.2.5",
-
-    "express": "^4.17.0",
     "@types/express": "^4.17.0",
-
+    "@types/mocha": "^5.2.5",
```
Comment (jcchavezs) [inline package.json]: This is just a re-sorting. Nothing important.

File: packages/zipkin/rollup.config.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import replace from 'rollup-plugin-replace';
 import {terser} from 'rollup-plugin-terser';
 import globals from 'rollup-plugin-node-globals';
 import builtins from 'rollup-plugin-node-builtins';
+import del from 'rollup-plugin-delete';
```
Comment (jcchavezs) [inline packages/zipkin/rollup.config.js]: ^ This is the plugin doing the trick for the clean up of old build files.

Comment (adriancole) [reply packages/zipkin/rollup.config.js]: cool

**Comment by adriancole** [comment]
lgtm on green

**Pull request closed without merging**